packages/blade/src/components/Box/BaseBox/baseBoxStyles.ts

```diff
@@ -213,14 +237,37 @@ const getAllProps = (
     left: getSpacingValue(props.left, props.theme, breakpoint),
 
     // Visual props
-    backgroundColor: getBackgroundValue(props.backgroundColor, props.theme, breakpoint),
+    backgroundColor: getColorValue(props.backgroundColor, props.theme, breakpoint),
     borderRadius: getBorderRadiusValue(props.borderRadius, props.theme, breakpoint),
     lineHeight: getSpacingValue(props.lineHeight, props.theme, breakpoint),
     border: getResponsiveValue(props.border, breakpoint),
     borderTop: getResponsiveValue(props.borderTop, breakpoint),
     borderRight: getResponsiveValue(props.borderRight, breakpoint),
     borderBottom: getResponsiveValue(props.borderBottom, breakpoint),
     borderLeft: getResponsiveValue(props.borderLeft, breakpoint),
+    borderWidth: getBorderWidthValue(props.borderWidth, props.theme, breakpoint),
+    borderColor: getColorValue(props.borderColor, props.theme, breakpoint),
+    borderTopWidth: getBorderWidthValue(props.borderTopWidth, props.theme, breakpoint),
+    borderTopColor: getColorValue(props.borderTopColor, props.theme, breakpoint),
+    borderRightWidth: getBorderWidthValue(props.borderRightWidth, props.theme, breakpoint),
+    borderRightColor: getColorValue(props.borderRightColor, props.theme, breakpoint),
+    borderBottomWidth: getBorderWidthValue(props.borderBottomWidth, props.theme, breakpoint),
+    borderBottomColor: getColorValue(props.borderBottomColor, props.theme, breakpoint),
+    borderLeftWidth: getBorderWidthValue(props.borderLeftWidth, props.theme, breakpoint),
+    borderLeftColor: getColorValue(props.borderLeftColor, props.theme, breakpoint),
+    borderTopLeftRadius: getBorderRadiusValue(props.borderTopLeftRadius, props.theme, breakpoint),
+    borderTopRightRadius: getBorderRadiusValue(props.borderTopRightRadius, props.theme, breakpoint),
+    borderBottomRightRadius: getBorderRadiusValue(
+      props.borderBottomRightRadius,
+      props.theme,
+      breakpoint,
+    ),
+    borderBottomLeftRadius: getBorderRadiusValue(
+      props.borderBottomLeftRadius,
+      props.theme,
+      breakpoint,
+    ),
+    borderStyle: hasBorder ? 'solid' : undefined,
```

Comment:
I think we should also make this a prop since we're anyway giving the control to the user

Reply:
Can't forsee any use-case for borderStyle yet. if the use case does come up we can expose it and change this to 
```
    borderStyle: borderStyle ?? hasBorder ? 'solid' : undefined,
```